include/revng/ABIAnalyses/Common.h

```diff
@@ -0,0 +1,213 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/SmallSet.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/GlobalVariable.h"
+#include "llvm/IR/Instruction.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Value.h"
+#include "llvm/Support/Casting.h"
+
+#include "revng/BasicAnalyses/GeneratedCodeBasicInfo.h"
+#include "revng/MFP/MFP.h"
+#include "revng/Model/Binary.h"
+#include "revng/StackAnalysis/StackAnalysis.h"
+#include "revng/Support/revng.h"
+
+namespace ABIAnalyses {
+
+using llvm::BasicBlock;
+using llvm::CallInst;
+using llvm::GlobalVariable;
+using llvm::Instruction;
+using llvm::LoadInst;
+using llvm::SmallSet;
+using llvm::SmallVector;
+using llvm::StoreInst;
+using llvm::Value;
```

Comment:
You're pulling all of these into the `ABIAnalyses` namespace.
People could use `ABIAnalyses::BasicBlock`, not sure this is what we want.

Reply:
definitely not what I wanted, removed and added `using namespace llvm;` in some function bodies when needed